posthog/async_migrations/setup.py

```diff
@@ -33,18 +38,51 @@ def reload_migration_definitions():
 reload_migration_definitions()
 
 
+def is_fresh_install(first_migration: str):
+    # To verify we're on a fresh install
+    # (1) check that the first migration is not started
+    # (2) check that there are no events in the events table (i.e. someone didn't try to upgrade to 1.40 from before async migrations introduction)
+    migration_instance = AsyncMigration.objects.filter(name=first_migration).first()
+    if migration_instance.status != MigrationStatus.NotStarted:
+        return False
+    event_cnt = sync_execute("SELECT count() FROM events")[0][0]
```

Comment:
This feels fragile as someone might sign up for only session recordings conceivably.

Reply:
Note that we only have this check for folks who are trying to upgrade from before async migrations were introduces to past this commit (i.e. before 1.30'ish -> 1.40), because they'd potentially have the first migration as NotStarted, but not be a fresh install. I don't believe we had anyone just use session recordings before that, that said I could change this to check for persons instead which covers both.

Reply:
An alternative I can think of is making the first CH migration add something that indicates, when the fresh install happened (or that it was before 1.40 if None).